### DISCUSSION TITLE
HTML Purifier

### DISCUSSION THREAD

**Discussion by Judx** [OpenMage/magento-lts#3600] [Ideas]
### Description (*)

Adding [HtmlPurifier](https://github.com/ezyang/htmlpurifier)

### Expected behavior (*)

HTML Purifier is an HTML filtering solution that uses a unique combination of robust whitelists and aggressive parsing to ensure that not only are XSS attacks thwarted, but the resulting HTML is standards compliant.

### Benefits

Improve security

### Additional information

[Magento Open Source 2.4.5](https://experienceleague.adobe.com/docs/commerce-operations/release/notes/magento-open-source/2-4-5.html?lang=en) added HtmlPurifier library to improve the MaliciousCode filter.

**Comment by Judx** [comment]
If this is a good idea I can do a pr

**Comment by addison74** [comment]
Please implement a PR then we can evaluate it by testing.

**Comment by Judx** [comment]
https://github.com/OpenMage/magento-lts/pull/3606
